web/concrete/src/Page/Page.php

```diff
@@ -2836,6 +2836,7 @@ public function add($pt, $data, $template = false)
             $pkgID = 0;
         }
 
+        $data['name'] = null;
```

Comment:
That canonical url array syntax will break in PHP 5.3, and this is in the
develop branch.

Honestly, can we just start making these changes against the version 8
branch, and treat this as the current 5.7.5.x branch, as I'm sure we're
going to be doing more bug fixes that we want to roll out to 5.7.5.x and
these changes are both less important and can potentially introduce new
bugs.

On Wed, Feb 3, 2016 at 9:03 AM, Korvin Szanto <notifications@github.com>
wrote:

> In web/concrete/src/Page/Page.php
> <https://github.com/concrete5/concrete5/pull/3474#discussion_r51752063>:
>
> > @@ -2836,6 +2836,7 @@ public function add($pt, $data, $template = false)
> >              $pkgID = 0;
> >          }
> >
> > +        $data['name'] = null;
>
> $data['name'] = $data['name'] ?? null; would be better here since data is
> passed in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/concrete5/concrete5/pull/3474/files#r51752063>.
>